### ISSUE TITLE
Current namespace is not displaying

### ISSUE THREAD

**Issue by bryanl** [vmware-tanzu/octant#676]
**What steps did you take and what happened:**
Current namespace is not displayed.

**Anything else you would like to add:**

<img width="914" alt="Screen Shot 2020-02-12 at 5 00 47 PM" src="https://user-images.githubusercontent.com/240/74381469-82fb0980-4db9-11ea-8526-47a6c9009b3c.png">

**Environment:**

```
Version:  (dev-version)
Git commit:  253bdba9
Built:  2020-02-12T21:55:29Z
```

**Comment by bryanl** [comment]
As a follow-up, it works in dark mode, so this is likely a CSS error.

**Comment by wwitzel3** [comment]
This was fixed #674
